api/pkg/transformer/executor/transformer.go

```diff
@@ -80,7 +80,7 @@ func (st *standardTransformer) Execute(ctx context.Context, requestBody types.Pa
 	if env.IsPreprocessOpExist() {
 		preprocessOut, err = env.Preprocess(ctx, requestBody, requestHeaders)
 		if err != nil {
-			return generateErrorResponse(err), err
+			return generateErrorResponse(err), nil
```

Comment:
why return nil?

Reply:
i think this one honest mistake, I'll revert this

Reply:
Shouldn't it be nil? The simulation response is error but the api response should still be successful right?

Reply:
yeah just realized this, that is trigger alert in ds-critical, I removed the error from the interface completely